nbgrader/plugins/export.py

```diff
@@ -9,6 +9,12 @@ class ExportPlugin(BasePlugin):
 
     to = Unicode("", help="destination to export to").tag(config=True)
 
+    student = List([], 
+        help="list of students to export").tag(config=True)
+    
+    assignment = Unicode("", 
```

Comment:
Why have two different ways of passing in lists here---a `List` in the case of students and then a `Unicode` string in the case of assignments? I think it would be better to have whichever method of passing in a sequence here be consistent.

Reply:
The mix in lists was because of the presence of number-only names (studentID's) and alpha-numeric (assignments). Now implemented a list only version which does require alpha-numeric names quoted (bash shell) and the quotes need to be escaped. 
Changed to List for both student and assignment in new commit.